spec/models/user_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe User, :type => :model do
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)